src/entities/attachment.rs

```diff
@@ -20,15 +20,13 @@ pub struct Attachment {
     /// (only present on local images)
     pub text_url: Option<String>,
     /// Meta information about the attachment.
-    #[serde(deserialize_with="empty_as_none")]
+    #[serde(deserialize_with = "empty_as_none")]
     pub meta: Option<Meta>,
     /// Noop will be removed.
     pub description: Option<String>,
 }
 
-fn empty_as_none<'de, D: Deserializer<'de>>(val: D)
-    -> Result<Option<Meta>, D::Error>
-{
+fn empty_as_none<'de, D: Deserializer<'de>>(val: D) -> Result<Option<Meta>, D::Error> {
```

Comment:
Please remove this change, as a policy if a function signature is longer than 80 characters I place the return type, where clause and starting parens each on a new line.